pkg/services/accesscontrol/accesscontrol.go

```diff
@@ -8,7 +8,7 @@ import (
 
 type AccessControl interface {
```

Comment:
Silly comment, but I would call the interface AccessController

Reply:
My two cents input -> `AccessControl` name comes with good enough semantic information and gives a clear message about the single concept it wants to encapsulate, which is "everything you need as a client to work with the access control system". `AccessController` is on the other side is more confusing (and overused, e.g. good old MVC...) term, e.g. it is "I know how to control things and when to control", which is too specific in our case as we want to be open to extension and expansion.